blocks/filter.py

```diff
@@ -55,6 +56,7 @@ class VariableFilter(object):
         A regular expression for the variable name. The Blocks name (i.e.
         `x.tag.name`) is used.
     application : :class:`.Application` instance
```

Comment:
Could you please rename it to `application` as it supports multiple arguments now? You should also add documentation for new options, such as `name_re`, `one` and change the `name` docstring.